src/index.js

```diff
@@ -179,6 +181,16 @@ const setLang = (lng) => {
           prefectureTrajectoryChart,
           LANG
         );
+        PrefectureTable.drawAllPrefectureTable(
```

Comment:
We previously didn't need to redraw the table here. What is the reason we need to draw the tables here now?

Reply:
I see now why you need to do this, to format the numbers to the right language on language change. Looks good.

Reply:
Yes, the number format may change when the language changes, and (unlike with translations) there are no original numbers nor hooks kept around, so they just have to be re-rendered.